packages/datadog-plugin-graphql/test/index.spec.js

```diff
@@ -987,13 +988,29 @@ describe('Plugin', () => {
         it('should support multiple executions on a pre-parsed document', () => {
           const source = `query MyQuery { hello(name: "world") }`
           const document = graphql.parse(source)
-
           expect(() => {
             graphql.execute({ schema, document })
             graphql.execute({ schema, document })
           }).to.not.throw()
         })
 
+        it('should not fail without directives in the document ' +
+          'and with subscription to datadog:graphql:resolver:start', () => {
+          const source = `query MyQuery { hello(name: "world") }`
+          const document = graphql.parse(source)
+          delete document.definitions[0].directives
+          delete document.definitions[0].selectionSet.selections[0].directives
+
+          function noop () {}
+          dc.channel('datadog:graphql:resolver:start').subscribe(noop)
+
+          expect(() => {
+            graphql.execute({ schema, document })
+          }).to.not.throw()
+
+          dc.channel('datadog:graphql:resolver:start').unsubscribe(noop)
```

Comment:
This will never run if the expectation fails.

Reply:
true, I'll add it in a finally